conda_smithy/configure_feedstock.py

```diff
@@ -1873,15 +1809,9 @@ def _azure_specific_setup(jinja_env, forge_config, forge_dir, platform):
         ],
     }
     if forge_config["azure"]["store_build_artifacts"]:
-        platform_templates["linux"].append(
-            ".scripts/create_conda_build_artifacts.sh"
-        )
-        platform_templates["osx"].append(
-            ".scripts/create_conda_build_artifacts.sh"
-        )
-        platform_templates["win"].append(
-            ".scripts/create_conda_build_artifacts.bat"
-        )
+        platform_templates["linux"].append(".scripts/create_conda_build_artifacts.sh")
+        platform_templates["osx"].append(".scripts/create_conda_build_artifacts.sh")
+        platform_templates["win"].append(".scripts/create_conda_build_artifacts.bat")
```

Comment:
I'd be especially glad to not split such trivial statements across multiple lines, which just makes it that much harder to digest.